docs/config.rst

```diff
@@ -67,4 +67,31 @@ All configuration options above in the general ``[gcp]`` may be used here. Addit
     For the Pub/Sub plugin, ``keyfile`` is not required when running against the `Pub/Sub Emulator <https://cloud.google.com/pubsub/docs/emulator>`_ that Google provides.
 
 
+gcp.gce
+~~~~~~~
+
+All configuration options from the general ``[gcp]`` section may be used here.
```

Comment:
So there's no config that would be different between GCE and GCRM APIs? Is it a conscious decision to make folks use one keyfile for both APIs?

Reply:
This is the "simple defaults" approach because both clients can share the same keyfile and scopes and work fine and I think the multi-keyfile approach could be premature at this point.